src/snowflake/snowpark/modin/pandas/dataframe.py

```diff
@@ -2347,15 +2347,15 @@ def squeeze(self, axis: Axis | None = None):
         len_columns = self._query_compiler.get_axis_len(1)
         if axis == 1 and len_columns == 1:
             return Series(query_compiler=self._query_compiler)
-        # get_axis_len(0) results in a sql query to count number of rows in current
-        # dataframe. We should only compute len_index if axis is 0 or None.
-        len_index = len(self)
-        if axis is None and (len_columns == 1 or len_index == 1):
-            return Series(query_compiler=self._query_compiler).squeeze()
-        if axis == 0 and len_index == 1:
-            return Series(query_compiler=self.T._query_compiler)
-        else:
-            return self.copy()
+        if axis in [0, None]:
+            # get_axis_len(0) results in a sql query to count number of rows in current
```

Comment:
i expect this should impact existing tests and see a query count change, if not, can you add a test for the case to make sure the query count is expected?

Reply:
Yes I will add a test, I wanted to see what it impacted first 😄

Reply:
Several tests failed, I have fixed them.